ROMFS/px4fmu_common/init.d-posix/1035_plane_ctrlalloc

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+#
+# @name Plane SITL
+#
+
+param set CA_AIRFRAME 3
```

Comment:
@jlecoeur For some reason I got segfaults when not doing this, need to investigate further.

Reply:
Yes there is a bug in the effectiveness update function that I have to fix.